NEWS.md

```diff
@@ -8,6 +8,8 @@ for demonstration purposes (#240).
 - Polars is internally moving away from string errors to a new error-type called `RPolarsErr` both on rust- and R-side. Final error messages should look very similar (#233).
 - LazyFrame_columns, _schema, _dtypes implemented. Improvements to internal `RPolarsErr`. Also `RPolarsErr` will now print each context of the error on a separate line (#250).
 - Add helpful reference landing page at `polars.github.io/reference_home` (#223, #264).
+- rust-polars' `simd` feature is now disabled by default. To enable it, set the environment variable
+  `RPOLARS_ALL_FEATURES` to `true` when build r-polars (#262).
```

Comment:
It's a bit unclear to me how this will work in practice. If I'm a classic user, do I have to do anything when I install `r-polars` binaries? Same question for a developer

Reply:
This is an update on the Rust crate build. If we are installing binary R package, this is irrelevant.

I will update the documentation on this in a follow-up PR as discussed here https://github.com/pola-rs/r-polars/issues/80#issuecomment-1610339106.